packages/backend-auth/src/userpool_access_policy_factory.ts

```diff
@@ -80,6 +80,13 @@ const iamActionMap: IamActionMap = {
     'cognito-idp:AdminAddUserToGroup',
     'cognito-idp:AdminRemoveUserFromGroup',
   ],
+  manageGroups: [
```

Comment:
@josefaidt how is this going to work with the groups defined as part of `defineAuth`?

Reply:
we should be safe, https://github.com/renebrandel/amplify-gen2-real-time-multi-group/blob/main/amplify/auth/resource.ts

Reply:
It certainly _will_ work. It's more a question of documentation / expectations. ie if you define `groups: ['admin']` and then delete the admin group in a lambda, your auth rules for that group are obviously going to stop working.

You also won't be able to configure group permissions for a group created dynamically because the backend definition has no idea about that group.

We probably need to document that groups can either be managed statically in the backend definition or dynamically in a lambda, but you shouldn't mix and match management strategies.

Reply:
We will need to document the "static" and "dynamic" group management workflows and caveats. While we won't be able to define auth rules for the dynamic groups, those can be stored in a field on the model and used with an auth rule. It'll be a different approach to how you use auth rules in a schema but we can document it

>  ie if you define groups: ['admin'] and then delete the admin group in a lambda, your auth rules for that group are obviously going to stop working

this is going to need to be in a callout at the top of the docs page